### PR TITLE
DF-123: Sentry integration amends

### PR DESCRIPTION
**Pull request by ababic**

### CODE REVIEW

File: config/versioning.py [hunk 1]
```diff
@@ -2,12 +2,19 @@
...
 
     head_path = os.path.join(base_dir, ".git", "HEAD")
 
-    with open(head_path, "r") as fp:
-        head = fp.read().strip()
+    try:
+        with open(head_path, "r") as fp:
+            head = fp.read().strip()
+    except FileNotFoundError:
+        return ""
```
Comment (TNA-Allan) [inline config/versioning.py:17]: Could we mark this as temporary or maybe  get_git_sha?

**Review by TNA-Allan — COMMENTED** [review]
One query, otherwise all looks good

**Review by TNA-Allan — COMMENTED** [review]
One query, otherwise all looks good

File: config/versioning.py (hunk 1 above)
Comment (ababic) [reply config/versioning.py:17]: There's no way to do this lazily, unfortunately - as the config is needed to initialise the sentry client. I think leaving it as a blank string if it cannot be identified is a safe enough fallback.

Perhaps when we move to AWS, there's be something similar we can get from the environment too.

**Pull request merged**